.gitignore

```diff
@@ -1,7 +1,10 @@
+data
```

Comment:
I think this should be /data, and we can also remove the /data/* subdirectory lines below.